### ISSUE TITLE
N_SIMPLE_CHEMISTRY_REACTION=2 gives incorrect HRR

### ISSUE THREAD

**Issue by rmcdermo** [firemodels/fds#6641]
@mcgratta Run the attached input file with NSCR=1 and 2 and look at the _hrr file.  With the 2 step scheme I am seeing HRR about 67% too high.

[test.fds.txt](https://github.com/firemodels/fds/files/2199150/test.fds.txt)

**Comment by rmcdermo** [comment]
So... HOC seem to be computed correctly as shown in the .out file.  But MLR_FUEL is 3.35e-4 for the 2 step case and 2.00e-4 (correct) for the 1 step case.  Somehow, HRRPUA is not getting converted correctly for 2 step.

**Comment by rmcdermo** [comment]
Here is the culprit:
```
IF (SF%HRRPUA>0._EB) THEN
         RN => REACTION(1)
         SF%MASS_FLUX(RN%FUEL_SMIX_INDEX) = SF%HRRPUA/RN%HOC_COMPLETE
         print *,RN%HOC_COMPLETE,SF%MASS_FLUX(RN%FUEL_SMIX_INDEX)
```
The "complete" HOC should account for the 2nd reaction.

**Comment by rmcdermo** [comment]
This is fixed in PR #6642.
